### DISCUSSION TITLE
Lost overview camera after VIM

### DISCUSSION THREAD

**Discussion by tomeberg** [Mr-MIBonk/M.I.B._More-Incredible-Bash#534] [Q&A]
I have a C7.5 with MIB high and BOSE. 
After I installed M.I.B. And activated the VIM function I've lost my 360 view.
To be precise I still have the overview option and the split screen but the camera is just black.
When I press the parking sensor button to exit the screen it will show the overview camera for about .5 Seconds before the screen goes back to previous menu.
Have tried to compare coding and adaption on 5f from an old backup, but nothing sticks out as relevent to my problem.
Anyone with an idea?

**Comment by Pe2Ce** [comment]
same

**Comment by tomeberg** [comment]
@Pe2Ce , it changes the view mode. Just need to activate it in the mmi menu.
